woocommerce/woocommerce-ios
# [Mobile Payments] Enable Moose on the Loose for all merchants

Closes: #6952 


### Description


Removes both the feature flag and experimental toggle for In-Person Payments in Canada, enabling the feature for everyone.

### Testing instructions


Go to Settings -> In-Person Payments and verify the onboarding shows the right state:

- US: available for both WCPay and Stripe extension
- Canada: available for WCPay but not Stripe
- Other countries: unavailable

You can also edit the scheme in Xcode to test on a `Release` build, which should match the configuration in the App Store:
![Screen Shot 2022-05-26 at 11 59 30](https://user-images.githubusercontent.com/8739/170465794-6b96a494-7b9a-4402-bb63-99c847a7deb7.png)



---
